cfme/scripting/bz.py

```diff
@@ -224,6 +225,15 @@ def list(directory, bz_status):
 
 @main.command(help="Set QE test coverage flag based on automates/coverage metadata")
 @click.argument("directory", default="cfme/tests")
+@click.option(
+    "-m",
+    "--inc_man",
```

Comment:
@prichard77 for consistency with the `pytest` arg, I suggest we just call this `--include-manual`

Reply:
Done